docs/getting_started.rst

```diff
@@ -97,6 +100,16 @@ maximum number of connections to save in the pool.
 Note: Idle connections within the pool may be terminated by the server, so will not remain open
 indefinitely meaning that this will not completely remove the overhead of creating new connections.
 
+Using library in app server environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This library can be used in an app server e.g. django and flask. When connecting
+to ``Cloudant`` or ``CouchDB``, you have the option of using the ``connect=True``
+argument during ``client`` construction or by calling ``client.connect()``.
+If the app server requires automatic renewal of the cookie authentication, add
```

Comment:
how does the user find out if an app server does or doesn't require automatic renewal of the cookie authentication?